src/NuGet.Clients/VisualStudio/IPackageManagerProvider.cs

```diff
@@ -0,0 +1,47 @@
+﻿using System.Runtime.InteropServices;
+using System.Threading;
+using System.Threading.Tasks;
+
+namespace NuGet.VisualStudio
+{
+    /// <summary>
+    /// Interface allowing integration of alternate package manager suggestion for a NuGet package. 
+    /// For example jQuery may appear on Bower and it might be more appropriate to install a package from Bower for certain projects 
+    /// </summary>
+    [ComImport]
+    [Guid("BCED5BF2-40FC-4D9F-BF0A-43CD4E9FF65F")]
+    public interface IPackageManagerProvider
```

Comment:
This should be IVsPackageManagerProvider

Our truly public VS interfaces have IVs on them.